### PR TITLE
IDEA-199921 Revert commit d8423f0 after commit b6a57a9

### PR DESCRIPTION
**Pull request by dahlstrom-g**
@develar?

### CODE REVIEW

File: platform/platform-impl/src/com/intellij/openapi/wm/impl/ToolWindowManagerImpl.java [hunk 1]
```diff
@@ -152,7 +152,7 @@ public void projectOpened(@NotNull Project project) {
         PropertyChangeListener focusListener = it -> myUpdateHeadersAlarm.request();
         KeyboardFocusManager keyboardFocusManager = KeyboardFocusManager.getCurrentKeyboardFocusManager();
         keyboardFocusManager.addPropertyChangeListener("focusOwner", focusListener);
-        Disposer.register(ToolWindowManagerImpl.this, () -> keyboardFocusManager.removePropertyChangeListener(focusListener));
+        Disposer.register(ToolWindowManagerImpl.this, () -> keyboardFocusManager.removePropertyChangeListener("focusOwner", focusListener));
```
Comment (dahlstrom-g) [inline platform/platform-impl/src/com/intellij/openapi/wm/impl/ToolWindowManagerImpl.java]: It looks like this was done by @fastasturtle in commit b6a57a9.

**Comment by fastasturtle** [comment]
Oh,  sorry, I wasn't aware of this activity. Yes, I'd say calling `request` on a disposed alarm is a pre-condition violation (and "assertion failed" is an expected result of it), and `removePropertyChangeListener` fix should be enough.

Let's wait for @develar to merge the revert back, though, as he owns the code.

**Comment by dahlstrom-g** [comment]
@develar?

**Comment by develar** [comment]
Does this extra check leads to any runtime issues? Or because this check hides  real issues like "and "assertion failed" is an expected result of it)"?

**Comment by dahlstrom-g** [comment]
I'm not aware of any problems encountered yet but I get the impression it's not what should be done in principle.

**Pull request closed without merging**